cluster/centos/config-default.sh

```diff
@@ -120,7 +120,7 @@ export FLANNEL_NET=${FLANNEL_NET:-"172.16.0.0/16"}
 
 # Admission Controllers to invoke prior to persisting objects in cluster
 # If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
```

Comment:
Can we add documentation that ValidatingWebhook should only be followed by ResourceQuota as nothing which modifies the object can come after it?

Reply:
That's guaranteed by the server code. The mutating admissions and non-mutating ones implements different interfaces and the non-mutating ones always run after the mutating ones.